android/capacitor/src/main/java/com/getcapacitor/plugin/SystemBars.java

```diff
@@ -123,7 +123,17 @@ private void setupSafeAreaInsets(boolean hasFixedWebView, boolean hasMetaViewpor
             }
 
             Insets safeArea = insets.getInsets(WindowInsetsCompat.Type.systemBars() | WindowInsetsCompat.Type.displayCutout());
-            injectSafeAreaCSS(safeArea.top, safeArea.right, safeArea.bottom, safeArea.left);
+            Insets imeInsets = insets.getInsets(WindowInsetsCompat.Type.ime());
+            boolean keyboardVisible = insets.isVisible(WindowInsetsCompat.Type.ime());
+
+            int bottomInsets = safeArea.bottom;
+
+            if (keyboardVisible) {
+                // Remove when https://issues.chromium.org/issues/457682720 is fixed
+                bottomInsets = imeInsets.bottom - bottomInsets;
+            }
```

Comment:
I'm not sure if this is something that should be removed when that issue is fixed, as that would effectively break it again for users with an older webview